framework/x509.py

```diff
@@ -98,42 +105,56 @@ def __hash(self):
             return hashes.SHA384()
         elif self.sign_alg == 'sha512':
             return hashes.SHA512()
-        else:
-            raise NotImplementedError("Not implemented hash algorithm: %s"
-                                      % self.sign_alg)
+        raise NotImplementedError(
+            'Not implemented hash algorithm: {0}'.format(self.sign_alg),
```

Comment:
f'Not implemented hash algorithm: {self.sign_alg}'

Reply:
Explanation not to use `f`-strings [from linter docs](https://wemake-python-stylegui.de/en/latest/pages/usage/violations/consistency.html#wemake_python_styleguide.violations.consistency.FormattedStringViolation)